### ISSUE TITLE
Update HLS Labs from Vivado HLS 2018 to Vitis 2024.1

### ISSUE THREAD

**Issue by nightt5879** [Xilinx/xup_high_level_synthesis_design_flow#7]
Hello!

I am a student currently learning FPGA as part of my coursework. Our instructor places a lot of emphasis on hands-on experiments, and we were assigned several labs from Xilinx's official website. This week, the task is focused on HLS (High-Level Synthesis). I found this repository while looking for resources, but I noticed that the tutorial uses Vivado HLS 2018.2.

Given that Xilinx is actively promoting the Vitis platform as the new synthesis tool, I was wondering if I could implement the labs using Vitis and submit four updated **_.md_** documents with HLS content, following the style of your original labs. You could review the changes and provide feedback, and I would be happy to make any improvements if needed.

I am raising this issue to ask if you would be interested in this PR, as I wouldn't want to submit a PR without knowing if it would be appreciated—I'd feel a bit disappointed if it got ignored.

Also, here's the link to the repository with six experiments I completed last week: 
https://github.com/nightt5879/FPGA
I'll be updating this repo with all future course-related projects, and there’s an English version in the README if you'd like to take a look.

Looking forward to your response!

**Comment by nightt5879** [comment]
emm.... i fork the respo and i found that the respo in here is the Vitis HLS,  i will try to using the new .md to start.If there have someting wrong i will come to ask 😂

**Comment by nightt5879** [comment]
emm.... i fork the respo and i found that the respo in here is the Vitis HLS(but Vitis HLS itself is planned to be deprecated in the future as Xilinx continues to evolve its development tools).So If this aligns with your roadmap, I would be happy to contribute by updating the labs, and I’m open to feedback if you have specific guidelines.

**Comment by joshual-xlnx** [comment]
HI @nightt5879  thanks, the latest version is 2023.2, not 2018.
There should be no big difference between 2023.2 and 2024.1
https://xilinx.github.io/xup_high_level_synthesis_design_flow/

**Comment by nightt5879** [comment]
> HI @nightt5879 thanks, the latest version is 2023.2, not 2018. There should be no big difference between 2023.2 and 2024.1 https://xilinx.github.io/xup_high_level_synthesis_design_flow/

Yes, I understand after forking and downloading it. However, my question is that when I open Vitis HLS, it shows a message indicating that the GUI may not be supported in future versions. Because of this, I used Vitis to run lab1, as demonstrated here: https://github.com/nightt5879/FPGA/blob/main/datasheet/lab1_vitis.pdf

**Comment by vickyiii** [comment]
Hi @nightt5879 ,

Thanks for your interest in this Git Repo. I have reviewed your updated Lab1 guide using Vitis HLS under Vitis Unified IDE.
• In v2024.1, running vitis_hls launches the AMD Vitis Unified IDE (when AMD Vitis tool is installed)
• To use the deprecated AMD Vitis HLS IDE, use vitis_hls –classic

**Comment by nightt5879** [comment]
Hi@vickyiii,
So do you think I could submit a PR about the process of running lab1 using Vitis? It would include an .md file and images, adding a lab1_vitis.md file under the docs folder and a lab1_vitis folder under the docs/images directory to store the related images for use in the .md file.

**Comment by nightt5879** [comment]
> Hi @nightt5879 ,
> 
> Thanks for your interest in this Git Repo. I have reviewed your updated Lab1 guide using Vitis HLS under Vitis Unified IDE. • In v2024.1, running vitis_hls launches the AMD Vitis Unified IDE (when AMD Vitis tool is installed) • To use the deprecated AMD Vitis HLS IDE, use vitis_hls –classic

So do you think I could submit a PR about the process of running lab1 using Vitis? It would include an .md file and images, adding a lab1_vitis.md file under the docs folder and a lab1_vitis folder under the docs/images directory to store the related images for use in the .md file.

**Comment by vickyiii** [comment]
Yes, there are more details on this PR operation. Please check out your Outlook e-mail box.
